src/arch/xtensa/Kconfig

```diff
@@ -8,4 +8,23 @@ config SMP
 	help
 	  Build SMP instead of UP
 
+config CORE_COUNT
+	int "Number of cores"
+	range 1 4
+	depends on CAVS
```

Comment:
can we have some defaults here? (generic default, not in defconfig)

Reply:
guess one core is always a must ;)

Reply:
Making it explicit is better. Also I meant that we need default not only for this config but for all 'int' configs.

Reply:
@mmaka1 Why does it depend on CAVS? Looks like generic config, if it's cavs-specific should it be named with CAVS_* prefix?

Reply:
@jajanusz I agree, on this level there is no relation to CAVS platforms. This is one of the knobs the architecture provides.